vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -28,6 +28,11 @@
     <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <IncludePkgdefInVSIXContainer>true</IncludePkgdefInVSIXContainer>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
```

Comment:
What's this change for?

Reply:
This was very unintended. It's not needed, will remove as soon as I'm at my pc again.